tests/components/rflink/test_sensor.py

```diff
@@ -141,15 +141,20 @@
 
     # test event for config sensor
     event_callback(
-        {"id": "test_alias_02_0", "sensor": "humidity", "value": 65, "unit": "%"}
+        {
+            "id": "test_alias_02_0",
+            "sensor": "humidity",
+            "value": 65,
+            "unit": UNIT_PERCENTAGE,
+        }
     )
     await hass.async_block_till_done()
 
     # test  state of new sensor
     updated_sensor = hass.states.get("sensor.test_02")
     assert updated_sensor
     assert updated_sensor.state == "65"
-    assert updated_sensor.attributes["unit_of_measurement"] == "%"
+    assert updated_sensor.attributes["unit_of_measurement"] == UNIT_PERCENTAGE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5093397)